completion_aggregator/serializers.py

```diff
@@ -259,22 +259,32 @@ def get_username(self, obj):
         """
         return obj.user.username
 
+    def _calculate_mean(self, obj):
+        """
+        Caclulate mean completion percent for all enrolled users.
+        """
+        enrollments = compat.get_users_enrolled_in(obj.course_key)
+        enrollment_count = enrollments.count()
+        if enrollment_count == 0:
+            return 0.
+
+        total = Aggregator.objects.filter(
+            course_key=obj.course_key,
+            aggregation_name='course',
+        ).aggregate(Sum('percent')).get('percent__sum', 0.)
```

Comment:
I'm not sure this is right.  I think if there are any nulls, the entire sum will become 0.  I suspect we need a more complex sql expression for the aggregate? Something like `Sum(Coalesce('percent', Value(0)))`.

[Database function documentation](https://docs.djangoproject.com/en/2.1/ref/models/database-functions/)

Reply:
OK! Didn't know that. Will update the PR and add a test for this.

Reply:
I've updated the code, but I don't think it's possible for percent to be null, so I'm not sure it's possible to test this.